client/src/styles/legacy/_CMSMain.scss

```diff
@@ -146,20 +146,6 @@
 		flex: 0 1 250px;
   }
 
-  .input-group-append {
-    z-index: 2;
-
-    &:last-of-type {
-      z-index: 1;
-    }
-
-    .btn {
-      margin-top: 0;
-      padding-top: $input-padding-y;
-      padding-bottom: $input-padding-y;
-    }
-  }
```

Comment:
https://getbootstrap.com/docs/5.0/migration/#forms

> Dropped `.input-group-append` and `.input-group-prepend`. You can now just add buttons and .input-group-text as direct children of the input groups.

Turns out the styling isn't needed anymore in any case, from what I can tell, so just removed this block outright.